doc/source/how-to/testing.rst

```diff
@@ -0,0 +1,536 @@
+Testing
+=======
+Unit and integration testing is critical for the successful continuous
+integration and delivery of any program or libraries belonging to the PyAnsys
+project.
+
+`Test Driven Development (TDD)`_ is the practice of writing unit tests before writing
+production code. This has the benefit of knowing that each of the new lines of
+code are working as soon as they're written. It's easier to track down problems
+as only a small amount of code has been implemented since the execution of the
+last test. Furthermore, all test cases do not have to be implemented at once but
+rather gradually as the code evolves TDD has been created by Kent Beck in the
+1990's as part of the Extreme Programming software development process
```

Comment:
```suggestion
`Test Driven Development (TDD)`_ is the practice of writing unit tests before writing
production code. The benefit of this practice is that you know each new line of
code is working as soon as it is written. It's easier to track down problems
because only a small amount of code has been implemented since the execution of the
last test. Furthermore, all test cases do not have to be implemented at once but
rather gradually as the code evolves. TDD was created by Kent Beck in the
1990s as part of the Extreme Programming software development process.
```